.expeditor/scripts/prep_and_run_tests.ps1

```diff
@@ -58,7 +58,6 @@ $env:Path = "$openssl_dir\bin;$ruby_dir\bin;" + $env:Path
 
 Write-Output "Configure bundle to build openssl gem with $openssl_dir"
 bundle config build.openssl --with-openssl-dir=$openssl_dir
-gem install openssl:3.3.0 -- --with-openssl-dir=$openssl_dir --with-openssl-include="$openssl_dir\include" --with-openssl-lib="$openssl_dir\lib"
```

Comment:
How is this related?

Reply:
I was getting failures in Windows unit and functional tests related to gem installs and gem versions. An analysis revealed that the version of openssl being installed on that line doesn't match the version of openssl we need and are using elsewhere. In short, it was creating version conflicts and because that was in the ps1 file, only the Windows nodes executing unit and functional tests were hitting it.

Reply:
But that's unrelated to GUIDs, that's just a bug in our testing framework, and should have it's own PR.

Reply:
Now moved and rebased. Thanks for the help